static/src/components/LoginView.vue

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+import { defineComponent } from "vue";
+import { LoginService as API } from "@/api/services/LoginService";
+
+export default defineComponent({
+  emits: ["login"],
+
+  data() {
+    return {
+      /**
+       * Content of the username filed
+       */
+      username: "",
+      /**
+       * Content of the password field
+       */
+      password: "",
+    };
+  },
+
+  methods: {
+    async loginUser() {
+      if (this.username != "" && this.password != "") {
+        try {
+          await API.login({
+            username: this.username,
+            password: this.password,
+          });
+          this.$emit("login");
+        } catch (ex) {
```

Comment:
Eine letzte Verständnisfrage hab ich noch: wo kommt dieses ex her? Oder geht der quasi automatisch in den catch-block wenn das API.login(...) einen Fehler zurückgibt?

Reply:
Hab noch einen commit hinzugefügt (hoffe es baut so noch 😅) der das vielleicht / hoffentlich auch etwas klarer macht?

Reply:
Hm ich glaub der change hats kaputt gemacht :D Lass das ruhig so wie vorher, war nur ne Verständnisfrage